diagnostic.md

```diff
@@ -5,28 +5,37 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    Visual Hierarchy:
+    ![visual hierarchy](http://emberigniter.com/images/distant-components/diagram.png)
+
+    Based on the url, the routing layer resolve a specific model to render the appropriate component.
+    The deeper the component is nested, the more levels of parents it needs to go up to send action to the parents of parents, etc and until it finally reach the route.
```

Comment:
This diagram does depict that.  Any pair of UI elements that are nested are a good example. ie., a list w/ list-items, a navbar w/ buttons, the sidebar of a blog w/ comments... etc.